spec/fixtures/cookbooks/oneview_test_api500_synergy/metadata.rb

```diff
@@ -0,0 +1,7 @@
+name             'oneview_test_api500_synergy'
+maintainer       'none'
```

Comment:
Perhaps this should be listed as `Hewlett Packard Enterprise`? As this is maintained by us at the moment.
The description of the field does imply that would be correct "The name of the person responsible for maintaining a cookbook, either an individual **or an organization.**"

Reply:
It doesn't matter, it is a fake testing cookbook, it doesn't even works